bark/models/behavior/behavior_rss/behavior_rss.cpp

```diff
@@ -89,7 +92,22 @@ Trajectory BehaviorRSSConformant::Plan(
   return last_traj;
 }
 
-
+void BehaviorRSSConformant::ApplyRestrictionsToNominalModel() {
+  #ifdef RSS
```

Comment:
this ifdef should be not in this function -> better CALL this function only when rss is defined. Thus calling it in Plan in the ifdef would be better